core/src/main/java/org/apache/accumulo/core/sample/impl/SamplerFactory.java

```diff
@@ -38,13 +42,16 @@ public static Sampler newSampler(SamplerConfigurationImpl config, AccumuloConfig
         clazz = ClassLoaderUtil.loadClass(context, config.getClassName(), Sampler.class);
 
       Sampler sampler = clazz.getDeclaredConstructor().newInstance();
-
+      sampler.validateOptions(config.getOptions());
       sampler.init(config.toSamplerConfiguration());
 
       return sampler;
 
     } catch (ReflectiveOperationException e) {
       throw new RuntimeException(e);
+    } catch (IllegalArgumentException | NullPointerException e) {
+      log.error("Cannot init sampler, invalid configuration: {}", e.getMessage(), e);
+      return null;
```

Comment:
It's a little weird that if the class can't be loaded, then it throws a runtime exception, but if it can't be started because of invalid config or some other problem, then it logs and returns null. I'm thinking these should probably be consistent. I would do something like:

```suggestion
    } catch (ReflectiveOperationException | RuntimeException e) {
      log.error("Cannot initialize sampler {}: {}", config.getClassName(), e.getMessage(), e);
      return null;
```

Reply:
Good idea, I like this change.